webpack/scenes/ContentViews/Details/Filters/CVErrataDateFilterContent.js

```diff
@@ -0,0 +1,250 @@
+  const [dateTypeSelectOpen, setDateTypeSelectOpen] = useState(false);
+  const [typeSelectOpen, setTypeSelectOpen] = useState(false);
+  const [selectedTypes, setSelectedTypes] = useState(types);
+  const dispatch = useDispatch();
+  const [activeTabKey, setActiveTabKey] = useState(0);
+
+  const onSave = () => {
+    dispatch(editCVFilterRule(
+      filterId,
+      {
+        id: rules[0]?.id,
+        content_view_filter_id: filterId,
+        start_date: startDate,
+        end_date: endDate,
+        types: selectedTypes,
+        date_type: dateType,
+      },
+      () => dispatch(getCVFilterDetails(cvId, filterId)),
+    ));
+  };
+
+  const resetFilters = () => {
+    setStartDate(rules[0]?.start_date || '');
+    setEndDate(rules[0]?.end_date || '');
+    setSelectedTypes(types);
+    setDateType(rules[0]?.date_type);
+  };
+
+  const onTypeSelect = (selection) => {
+    if (selectedTypes.includes(selection)) {
+      if (selectedTypes.length === 1) return;
+      setSelectedTypes(selectedTypes.filter(e => e !== selection));
+    } else setSelectedTypes([...selectedTypes, selection]);
+  };
+
+  const singleSelection = selection => (selectedTypes.length === 1
+    && selectedTypes.includes(selection));
+
+  const formatDate = (date, emptyText = null) => {
+    if (!date) return emptyText;
+    return new Date(date).toLocaleDateString('en-US', {
+      day: '2-digit',
+      month: '2-digit',
+      year: 'numeric',
+    });
+  };
+
+  const dateParse = (date) => {
+    if (date === '') return new Date();
+    const split = date.split('/');
+    if (split.length !== 3) {
+      return undefined;
+    }
+    const month = split[0];
+    const day = split[1];
+    const year = split[2];
+    return new Date(`${year}-${month.padStart(2, '0')}-${day.padStart(2, '0')}T00:00:00`);
+  };
+
+  const saveDisabled = () => (
+    (Number.isNaN(Date.parse(dateParse(startDate))) ||
+      Number.isNaN(Date.parse(dateParse(endDate)))
+    ) ||
+    (
+      isEqual(formatDate(rules[0]?.start_date), formatDate(startDate)) &&
+      isEqual(formatDate(rules[0]?.end_date), formatDate(endDate)) &&
+      isEqual(sortBy(types), sortBy(selectedTypes)) &&
+      isEqual(rules[0]?.date_type, dateType)
+    )
+  );
+
+  const dateFormat = date => `${(date.getMonth() + 1).toString().padStart(2, '0')}/${date.getDate().toString().padStart(2, '0')}/${date.getFullYear()}`;
+
+  useEffect(() => {
+    if (!repositories.length && showAffectedRepos) {
+      setActiveTabKey(1);
+    } else {
+      setActiveTabKey(0);
+    }
+  }, [showAffectedRepos, repositories.length]);
+
+  const tabTitle = (inclusion ? __('Included') : __('Excluded')) + __(' errata');
+
+  return (
+    <Tabs activeKey={activeTabKey} onSelect={(_event, eventKey) => setActiveTabKey(eventKey)}>
+      <Tab eventKey={0} title={<TabTitleText>{tabTitle}</TabTitleText>}>
+        <div className="tab-body-with-spacing">
+          <Form onSubmit={onSave}>
```

Comment:
@Andrewgdewar : The cause of all the params in the URL on submit. Removing this `onSubmit={onSave}` and `<Button type="submit">` resolves the whole page refresh issue on this page..I am assuming it's the same on other pages.

Reply:
It is..but it also adds the form data to the url and causes a whole page refresh..😕